tests/repository_test/repository_test.go

```diff
@@ -0,0 +1,59 @@
+package repository_test
+
+import (
+	"boilerplate-api/api/repository"
```

Comment:
could not import boilerplate-api/api/repository (no required module provides package "boilerplate-api/api/repository")